setup.py

```diff
@@ -4,21 +4,30 @@
 # For details on the licensing terms, see the LICENSE file.
 # SPDX-License-Identifier: MIT
 
-from setuptools import setup
+from setuptools import setup, find_packages
+from pathlib import Path
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
 
 setup(
-    name="PFDL Scheduler",
-    version="0.9.1",
-    description="Execution engine for Production Flow Description Language (PFDL) files.",
+    name="pfdl_scheduler",
+    version="0.9.0",
+    description="Parser and Scheduler of Production Flow Description Language (PFDL) files.",
+    long_description=long_description,
+    long_description_content_type='text/markdown',
     author="Peter Detzner, Maximilian Hoerstrup",
```

Comment:
General remark, do you want to refer to the copyright holders / the authors more generally instead of naming individual persons?

Reply:
I will do that!